docs/data/material/getting-started/support/support.md

```diff
@@ -8,15 +8,10 @@ Welcome to the MUI community!
 Our tools are used by thousands of developers and teams all around the world, many of whom actively engage with the community through various online platforms.
 These are the best places to start asking questions and looking for answers when you need help.
 
-### Discord
-
-MUI maintains an official [Discord server](https://mui.com/r/discord/) for real-time updates and discussions.
-This is a great place to post if you need help troubleshooting or you're looking for feedback on a project.
-
```

Comment:
I was thinking more about this page, and in hindsight, https://master--material-ui.netlify.app/material-ui/getting-started/support/ feels very strange, Discord is the worst place possible to get support.